### PR TITLE
add mtgstocks id and prices schema and documentation.

### LINKED ISSUE #156: New data added
New fields added w/ example (These are to each card object)

Future note: The dates are wrong right now, but they will be Sundays in all offical releases. And we will re-build on Sundays for deployment with a new version date of some kind. Still figuring out how I want to call the version#/date

```
"mtgstocksId": 40272,
"prices": {
                "paper": {
                    "2019-01-23": 0.25,
                    "2019-01-30": 0.25,
                    "2019-02-06": 0.25,
                    "2019-02-13": 0.25,
                    "2019-02-20": 0.26,
                    "2019-02-27": 0.26,
                    "2019-03-06": 0.27,
                    "2019-03-13": 0.29,
                    "2019-03-20": 0.27,
                    "2019-03-27": 0.27,
                    "2019-04-03": 0.28,
                    "2019-04-10": 0.29,
                    "2019-04-17": 0.29,
                    "2019-04-24": 0.28
                }
            },
```

Also added `mtgstocks` to purchaseUrl feed

### PR DESCRIPTION
**Pull request by staghouse**
## Changelog
- Add mtgstocks.com id and prices schema to card objects.

Close #156

### CODE REVIEW

File: docs/.vuepress/public/schemas/Prices.schema.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "paper": {
+    "type": "object(string: float)",
+    "example": "\"2019-04-28\": 0.25",
+    "description": "Paper format prices of a card."
```
Comment (ZeldaZach) [inline docs/.vuepress/public/schemas/Prices.schema.json]: Please add "Courtesy of [mtgstocks](https://www.mtgstocks.com?utm_campaign=mtgjson&utm_medium=mtgjson&utm_source=mtgjson)." After the description.

File: docs/structures/prices/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "enable": "true"
+  }
+}
+---
+
+# Prices
+
+Prices is a structure defining an object of card prices based on date.
+
+> Parent structure: [card](../card), [AllCards](../../all-cards)
```
Comment (ZeldaZach) [inline docs/structures/prices/README.md]: all-cards links wrong

**Comment by staghouse** [comment]
Updated.

File: docs/.vuepress/public/schemas/Prices.schema.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "paper": {
+    "type": "object(string: float)",
+    "example": "\"2019-04-28\": 0.25",
+    "description": "Paper format prices of a card. Courtesy of mtgstocks."
```
Comment (ZeldaZach) [inline docs/.vuepress/public/schemas/Prices.schema.json]: Do the URL trick we did with mtgsqlive please <3

**Pull request merged**